### PR TITLE
Uptake new verrazzano-monitoring-operator to fix dev profile install

### PR DESCRIPTION
**Pull request by ddrechse**

### CODE REVIEW

File: install/chart/templates/03-verrazzano-monitoring-operator.yaml [hunk 1]
```diff
@@ -394,7 +394,6 @@ spec:
             successThreshold: 1
             timeoutSeconds: 5
           args:
-            - --v=4
```
Comment (kminder) [inline install/chart/templates/03-verrazzano-monitoring-operator.yaml:397]: What was this required before but not now?

Comment (ddrechse) [reply install/chart/templates/03-verrazzano-monitoring-operator.yaml:397]: See Chen's logging changes, are checkins have become inermingled

Comment (ddrechse) [reply install/chart/templates/03-verrazzano-monitoring-operator.yaml:397]: without that, monitoring-operator fails to start

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**